apps/prairielearn/src/pages/partials/submission.ejs

```diff
@@ -73,7 +73,14 @@
           <% } %>
         </span>
       </div>
-      <small>Submitted at <%- submission.formatted_date %></small>
+      <span class="small">
+        <% if (authz_data.user.uid == submission.uid) { %>
```

Comment:
I'm wondering if this should be `authn_user` instead of `user`. In essence: if I use effective user, should it show "You" for the authenticated user or the effective user?

Reply:
Good point, this should probably be effective user.

Reply:
We're no longer showing "You"; we always just show the UID.